### ISSUE TITLE
Conversion prompt hides bottom bar 

### ISSUE THREAD

**Issue by avivapinchas** [ParabolInc/parabol#3759]
## Issue - Bug

With the new bottom bar, the conversion prompt hides the controls for the facilitator. 

![Screen Shot 2020-04-17 at 2 06 04 PM](https://user-images.githubusercontent.com/11621016/79607886-a55f1b00-80b9-11ea-81ec-cbe23423344f.png)

- **Estimated effort:** 11points ([see CONTRIBUTING.md](https://github.com/ParabolInc/parabol/blob/master/CONTRIBUTING.md#points-and-sizes))

### Acceptance Criteria (optional)
Meeting facilitators can:
   - See and interact with the bottom bar, even when they are also a billing leader being prompted to convert

**Comment by mattkrick** [comment]
@ackernaut what do you think about this? do we just make the snackbar pop up like 48px higher?

**Comment by ackernaut** [comment]
@mattkrick yeah, at least 72 px from the bottom? At the larger breakpoint, the bottom bar’s footprint is 8 + 56 then at least 8 above that?

Tangent: one thing that I still don’t like is how the bottom bar (and in this case the snackbar) is always centered in the viewport. In my mind I wish these components were centered in the main area when any sidebar is open (be it 1 side or both sides). I know we have client state that lets us know when the sidebar is open. That’s my bad idea for how to handle the centering.

**Comment by ackernaut** [comment]
@mattkrick we should also give thought to when there’s a snackbar and the confirmation tooltip, [see Figma](https://www.figma.com/file/n0V5H6PBU0Pf35ctWPlWol/Confirm-Ready-and-End-Meeting?node-id=249%3A0)
